README.md

```diff
@@ -473,6 +473,27 @@ XRayMiddleware(app, xray_recorder)
 db = XRayFlaskSqlAlchemy(app)
 
 ```
+
+### Ignoring httplib requests
+
+If you want to ignore certain httplib requests you can do so based on the hostname or URL that is being requsted. 
```

Comment:
Could you explicitly call out that this supports Unix-style regex or `fnmatch` regex (as opposed to `re`)? Also, can we add an example using subclass matching?

Reply:
@willarmiros updated to add both of those items.